### ISSUE TITLE
What are these items in the capture screen

### ISSUE THREAD

**Issue by professor99** [google/orbit#3619]
Hi all,

Firstly I'm using the windows version of orbit. 

I found it very useful in it's ability to hook functions to examine actual function timelines and function call count.

There presently is no help items or online tutorials on Orbit anywhere on the internet I can find either for the old windows version or the newer Linux version. The only help available is the rudimentary help on the README on Github.

I acknowledge that more help is in the pipeline as answers to the following issue reflects.

[https://github.com/google/orbit/issues/775]

I've worked out most of how to use orbit and the capture screen but I'm at a loss to figure out what the following two items are on the capture screen as illustrated in the attached image below. Looking thru the source which is sparsely documented I haven't been able to figure this out.

1. What is in this section and what do these boxes represent?

2. I've figured this box relates to a thread with the timeline of the hooked functions below. However what does the white lines in this as pointed to by the arrow represent?

Thanks
Andy

 
![image](https://user-images.githubusercontent.com/2804871/164960439-bd09861e-1e9a-48ea-a1d9-3614159f2b14.png)

**Comment by florian-kuebler** [comment]
Hi.
1. Is the "scheduling view". Each line is a CPU core. When there is a colored bar (timer), it means that the thread (with that color) was executed on that core. It _should_ be possible to hover over such a thread slice to see which thread it is exactly. Also clicking on the thread slice, _should_ highlight the corresponding thread track below.

2. Each white line is one callstack taken by sampling.

**Comment by professor99** [comment]
Thanks heaps for the reply Florian! 

It really helps knowing what these boxes are and how they can be used and makes Orbit even more useful. Hopefully my observances and questions and your answers will help other users besides me. After I posted this message I notice turning off some of the default options result in the boxes I described not been shown but I still couldn't figure this one out. I will paste later about these options to inform others.  

Cheers 
Andy
